pkg/controller/jmeter_integration_test.go

```diff
@@ -162,6 +162,7 @@ func kubeTestClient() clientSetV.Clientset {
 	}
 
 	config, err := BuildConfig()
+	config.Timeout = 60 * time.Second
```

Comment:
Why is this not set within `BuildConfig`? Seems like this could fix some issues with people forgetting to set the timeout.

Reply:
Apparently this is already set in `BuildConfig`, so this can be removed

Reply:
Done in 6b6ec81